src/ert/parsing/config_schema_deprecations.py

```diff
@@ -25,10 +26,17 @@
     *[
         DeprecationInfo(
             keyword=kw,
-            message=f"Using {kw} with substitution strings"
+            message=lambda line: f"Using {kw} with substitution strings"
```

Comment:
Verify this lambda, lint provides this warning;
`Function definition does not bind loop variable 'kw'.`

Reply:
Fixed